include/jemalloc/internal/private_symbols.txt

```diff
@@ -74,26 +74,6 @@ arena_tdata_get
 arena_tdata_get_hard
 arenas
 arenas_tdata_cleanup
```

Comment:
Actually, now that you mention it, no; this diff generated externally visible symbols. Fixed.